encord/utilities/client_utilities.py

```diff
@@ -27,6 +27,27 @@ def pretty_print(data):
 
 
 class APIKeyScopes(Enum):
+    """
+    The APIKeyScope is used to provide specific access rights to a project through
+    :meth:`.EncordUserClient.create_project_api_key`. The options are a follows:
+
+        * ``LABEL_READ``: access to
```

Comment:
Not sure that I like to say that this is not exhaustive, as this would be very vague. 
However, I do not see a good process for keeping this up to date. Perhaps this is good to discuss at the demo slot today.